src/enums/MeasureUnit.java

```diff
@@ -0,0 +1,13 @@
+package enums;
+
+public enum MeasureUnit {
```

Comment:
I think you can remove `KILOGRAMS`,  `MILIGRAMS`, `BOTTLE`, `CANS`.

I don't think you need this at all since you can convert these values from the exsiting ones. I don't see benefits on keeping these ones mentioned aboved

Reply:
solved